src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -38,6 +40,14 @@ public static void ______TS(String description) {
     }
     // CHECKSTYLE.ON:AbbreviationAsWordInName|MethodName
 
+    /**
+     * Set the required locale for tests.
+     */
+    @BeforeSuite
+    public void setLocale() {
+        Locale.setDefault(new Locale("en", "US"));
```

Comment:
Remove this?

Reply:
Let's have this any way to ensure that all parts of test being run are using this Locale?